src/v1/app.files.ts

```diff
@@ -247,7 +247,12 @@ export function initFileTree() {
         setEnabled(uiFiles.cloneKsyFile, isLocal && isKsy);
         setEnabled(uiFiles.deleteItem, isLocal);
         setEnabled(uiFiles.generateParser, isKsy);
-        uiFiles.fileTreeContextMenu.css({ display: "block", left: e.pageX, top: e.pageY });
+
+        uiFiles.fileTreeContextMenu.css({ display: "block", visibility: "hidden", "z-index": -1 }); // still invisible but has width & height
```

Comment:
Isn't setting `visibility` enough here?

Reply:
In the particular situation, the `fileTreeContextMenu` is not displayed on the page at all (it has [`display: none` because of the external styles](https://github.com/kaitai-io/kaitai_struct_webide/blob/2a967590d413e6995e04f096b7f2168cca374c6e/css/app.css#L26)). However, `display: none` turns off the display completely ([_"the document is rendered as though the element did not exist"_](https://developer.mozilla.org/en-US/docs/Web/CSS/display#Box)), so trying to get dimensions with `width()` and `height()` will yield `0`. So it's necessary to set `display` to something else than `none`.

And I was thinking: setting just `display: block` could in theory show the context menu on a wrong position for a very short time (until the calculation is done). That's why I'm setting `visibility: "hidden", "z-index": -1`. But it probably can't happen, because the JavaScript execution is apparently blocking the re-rendering (https://stackoverflow.com/a/6908240).

I'll remove these properties so they don't confuse other people in the future.